buildSrc/src/main/groovy/io.pivotal.cfenv.java-conventions.gradle

```diff
@@ -116,6 +116,12 @@ publishing {
                     def pomNode = asNode()
                     def dependencyManagementNode = pomNode.get('dependencyManagement')
                     if (dependencyManagementNode) pomNode.remove(dependencyManagementNode)
+
+                    if (project.name == 'java-cfenv-test-support') {
```

Comment:
Can this be done in the `java-cfenv-test-support` build.gradle instead of adding this special conditional here?

Reply:
I tried, even if it means duplicating the configuration and it fails with `Maven publication 'mavenJava' cannot include multiple components` failing on `from components.java`.